.gitignore

```diff
@@ -9,6 +9,9 @@ result
 *.snap*
 !cli/tests/cli-reference@.md.snap
 
+# mkdocs
```

Comment:
I'd say "uv and mkdocs to build the docs website"

Also, should we add `.python-version` to gitignore as well, or do newer versions of `uv` no longer generate it?

Reply:
`uv` does not create it unless you run `uv python pin`.

Reply:
Ah, makes sense, thanks! Maybe it's still nice to add for anybody who does run `uv python pin`? But yeah, it doesn't sound like it matters much whether we add `.python-version` here or not.

Reply:
As a Rust developer who barely touches Python, I've never needed to pin the Python version. I'm happy with adding `--python <version>` when creating virtual environments if I need a specific version.

Reply:
I’ll add it

Reply:
@mati865 There's almost no cost to adding it AFAICT, so we can do it even if it only benefits one person sometime.

Reply:
Sure, I don't have any problem with that. Just wanted to point out that file is not created unless the user requests it.